rust/src/detect/transform_base64.rs

```diff
@@ -80,6 +87,19 @@ impl SCDetectTransformFromBase64Data {
             ..Default::default()
         }
     }
+
+    pub fn serialize(&mut self, nbytes: &str, offset: &str) {
+        let mut r = Vec::new();
```

Comment:
we can compute the capacity needed before hands

Reply:
Yeah, I've seen lints trigger with a vec followed by some pushes where vec![] could have been used for the first few elements. Or a capacity.